crates/deno_task_shell/Cargo.toml

```diff
@@ -29,6 +29,7 @@ pest_derive = "2.7.12"
 dirs = "5.0.1"
 pest_ascii_tree = { git = "https://github.com/prsabahrami/pest_ascii_tree.git", branch = "master" }
 miette = "7.2.0"
+lazy_static = "1.4.0"
```

Comment:
I don't see this dependency used anywhere :)

Reply:
It is imported in parser.rs and used in line 533 to define precedence over ops.